pkg/cli/rm_helper.go

```diff
@@ -0,0 +1,207 @@
+package cli
+
+import (
+	"fmt"
+	"github.com/acorn-io/acorn/pkg/prompt"
+	"github.com/pterm/pterm"
+	"strings"
+
+	"github.com/acorn-io/acorn/pkg/client"
+	"github.com/spf13/cobra"
+)
+
+func addRmObject(rmObjects *RmObjects, obj string) {
+	switch strings.ToLower(obj) {
+	case "app":
+		rmObjects.App = true
+	case "container":
+		rmObjects.Container = true
+	case "secret":
+		rmObjects.Secret = true
+	case "volume":
+		rmObjects.Volume = true
+	case "a":
+		rmObjects.App = true
+	case "c":
+		rmObjects.Container = true
+	case "s":
+		rmObjects.Secret = true
+	case "v":
+		rmObjects.Volume = true
+	default:
+		pterm.Warning.Printf("%s is not a valid type\n", obj)
+	}
+}
+func getSecretsToRemove(arg string, client client.Client, cmd *cobra.Command) ([]string, error) {
+	var result []string
+	secrets, err := client.SecretList(cmd.Context())
+	apps, _ := client.AppList(cmd.Context())
+
+	if err != nil {
+		return nil, err
+	}
+
+	for _, secret := range secrets {
+		if arg == strings.Split(aliases(&secret, apps)[0], ".")[0] {
+			result = append(result, secret.Name)
+		}
+	}
+	return result, nil
+}
+func getVolumesToDelete(arg string, client client.Client, cmd *cobra.Command) ([]string, error) {
+	var result []string
+	volumes, err := client.VolumeList(cmd.Context())
+	if err != nil {
+		return nil, err
+	}
+
+	for _, volume := range volumes {
+		if arg == volume.Status.AppName {
+			result = append(result, volume.Name)
+		}
+
+	}
+	return result, nil
+}
+func getContainersToDelete(arg string, client client.Client, cmd *cobra.Command) ([]string, error) {
+	var result []string
+	containers, err := client.ContainerReplicaList(cmd.Context(), nil)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, container := range containers {
+		if arg == strings.Split(container.Name, ".")[0] {
+			result = append(result, container.Name)
+		}
+	}
+	return result, nil
+}
+func removeContainer(arg string, c client.Client, cmd *cobra.Command, f bool) error {
```

Comment:
Should this delete the `Deployment` associated with this "container"? Otherwise they continue to come back online. See my comment in `containers_rm.go`

Reply:
resolving, as above